core/integration/features/verified/Test-Target_Stake_Scaling.feature

```diff
@@ -0,0 +1,89 @@
+
+    And the average block duration is "1"
+
+    And the parties submit the following liquidity provision:
+      | id  | party  | market id | commitment amount | fee   | side | pegged reference | proportion | offset | lp type    |
+      | lp1 | party0 | ETH/MAR22 | 50000             | 0.001 | sell | ASK              | 500        | 17     | submission |
+      | lp1 | party0 | ETH/MAR22 | 50000             | 0.001 | buy  | BID              | 500        | 17     | amendment  |
+
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference  |
+      | party1 | ETH/MAR22 | buy  | 1      | 900   | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-1  |
+      | party1 | ETH/MAR22 | buy  | 1      | 990   | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-1  |
+      | party1 | ETH/MAR22 | buy  | 10     | 1000  | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-2  |
+      | party2 | ETH/MAR22 | sell | 10     | 1000  | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-3 |
+      | party2 | ETH/MAR22 | sell | 1      | 1010  | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-1 |
+      | party2 | ETH/MAR22 | sell | 1      | 1100  | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-2 |
+      | party3 | ETH/MAR22 | sell | 100    | 1010  | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-4 |
+
+    And the parties submit the following liquidity provision:
+      | id  | party  | market id | commitment amount | fee   | side | pegged reference | proportion | offset | lp type   |
+      | lp1 | party0 | ETH/MAR22 | 55000             | 0.001 | sell | ASK              | 500        | 17     | amendment |
+      | lp1 | party0 | ETH/MAR22 | 55000             | 0.001 | buy  | BID              | 500        | 17     | amendment |
+
+    And the parties cancel the following orders:
+      | party  | reference  |
+      | party3 | sell-ref-4 |
+
+    Then the opening auction period ends for market "ETH/MAR22"
+    And the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "ETH/MAR22"
+
+    And the market data for the market "ETH/MAR22" should be:
+      | mark price | trading mode            | horizon | min bound | max bound | target stake | supplied stake | open interest |
+      | 1000       | TRADING_MODE_CONTINUOUS | 3600    | 973       | 1027      | 9484         | 55000          | 10            |
+
+    Then debug detailed orderbook volumes for market "ETH/MAR22"
+
+    Then the parties should have the following margin levels:
+      | party  | market id | maintenance | search | initial | release |
+      | party0 | ETH/MAR22 | 79043       | 86947  | 94851   | 110660  |
+      | party3 | ETH/MAR22 | 0           | 0      | 0       | 0       |
+    And the parties should have the following account balances:
+      | party  | asset | market id | margin  | general   | bond  |
+      | party0 | USD   | ETH/MAR22 | 94851   | 499850149 | 55000 |
```

Comment:
@witgaw @Jiajia-Cui the party0 margin account is now different here since the recents changes from the core. Is that as you expect it?

Reply:
yes, the party0 margin account looks correct